src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2333,9 +2333,9 @@ class S4Importer final : public IParkImporter
 
     void FixEntrancePositions()
     {
-        for (sint32 i = 0; i < MAX_PARK_ENTRANCES; i++)
+        for (uint8 i = 0; i < MAX_PARK_ENTRANCES; i++)
```

Comment:
Since this is setting all entrances to null and the park entrance array is static, I'd suggest using `Util::CountOf(gParkEntrance)` here instead of `MAX_PARK_ENTRANCES`, and i can be of type `size_t`.